tests/admin_checks/tests.py

```diff
@@ -43,6 +45,17 @@ def check(self, **kwargs):
 )
 class SystemChecksTestCase(SimpleTestCase):
 
+    def setUp(self):
+        super(SystemChecksTestCase, self).setUp()
+        self.previous_sites = list(admin.sites.all_sites)
```

Comment:
We typically use an underscore prefix in a case like this: `self._previous_sites`